doc/documentation/src/analysis_guide_templates/preprocessing.rst

```diff
@@ -6,15 +6,28 @@ Preprocessing
 |FOURC| reads the mesh, boundary conditions, materials and simulation parameters from a central
 input file.
 
-.. admonition:: Under development
 
-    A large refactoring effort is currently in progress to improve |FOURC|'s input files.
-    The information in this section is not fully updated to reflect these changes.
+Working with |FOURC| input files
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-There are not so many means to create a valid input file. At this point, we know of the following
-different ways to create an input file. In general, you'll have two options:
+|FOURC| input files are text files in the YAML format. Since YAML is a superset of JSON,
+you may also use JSON syntax although we tend to prefer YAML for human-readability.
+By convention, |FOURC| input files have the extension ``.4C.yaml`` or ``.4C.json``.
 
-#. Either you create the mesh in |FOURC|'s native format directly,
+We build a schema called ``4C_schema.json`` which is located in the root build directory. This
+schema describes the structure of a valid |FOURC| input file.
+If you want to write and edit |FOURC| input files, be sure to set up schema validation in your editor
+(see :ref:`installation` for details). Doing so provides documentation and autocompletion
+for all parameters.
```

Comment:
This statement is not true. While autocompletion and partial validation are nice features, you can write and edit 4C input files without having scheme integration into your IDE.

I think we should hint at the possibility to do so, but not make it sound to be required.